pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md

```diff
@@ -560,4 +561,47 @@ Each basic element can have specific style rules defined for it, controlling how
 }
 ```
 
-For a list of Rich Messaging capabilities for each channel, browse or search the table on the [Knowledge Center](https://knowledge.liveperson.com/messaging-channels-messaging-channels-capabilities-comparison.html).
\ No newline at end of file
+### Element Accessibility Attributes
+
+Each basic element can have specific accessibility attributes defined to them. 
+Some of the widely used attributes are listed below. For the entire list of supported attributes, please refer to [this list](https://github.com/LivePersonInc/json-pollock/blob/master/js/schema/accessibilityWeb.json).
+
+#### Properties
+
+| Property Name    | Description                                                                 |
```

Comment:
Please switch to sentence case (“Property name”).

Reply:
@j9t I wanted it to be consistent with other elements at other places in the page so I put a capital N.
https://developers.liveperson.com/getting-started-with-rich-messaging-introduction.html#element-styling